app/src/main/java/de/grobox/transportr/trips/search/DirectionsFragment.java

```diff
@@ -197,6 +199,12 @@ public void onCreateOptionsMenu(Menu menu, MenuInflater inflater) {
 					})
 					.show();
 		}
+
+		Drawable drawable = menu.findItem(R.id.action_swap_locations).getIcon();
+		if (drawable != null) {
+			drawable.mutate();
+			drawable.setColorFilter(0xffffffff, PorterDuff.Mode.SRC_ATOP);
+		}
```

Comment:
I think we might not be using this icon elsewhere, so we could just set it to white in the XML.

Reply:
It is used when pressing the overflow menu for favorites

Reply:
Right, so we need to mutate here.